reana_server/rest/workflows.py

```diff
@@ -167,6 +185,10 @@ def get_workflows(user, **kwargs):  # noqa
                     launcher_url:
                       type: string
                       x-nullable: true
+                    owner_email:
```

Comment:
This maybe needs to be changed to `user_email` depending on what we decide in reana-workflow-controller